README.md

```diff
@@ -1,108 +1,138 @@
-    <version>1.0.0</version>
-</dependency>
-```
+    `./gradlew build`
 
-### Gradle
-```
-implementation 'engineering.gds-reliability:gds-metrics-dropwizard:1.0.0'
-```
+2. Publish to a [Maven][] local repository, by running:
 
-### Configure the app to be used by prometheus
+    `gradle publishToMavenLocal`
 
-1. Your application must be deployed on [PaaS](https://www.cloud.service.gov.uk/).
-2. Add the environment variable `PROMETHEUS_METRICS_PATH`. Example:
-```
-PROMETHEUS_METRICS_PATH=/metrics
-```
-3. If you want to have Dropwizard extended metrics enabled, define the `ENABLE_DROPWIZARD_METRICS` environment variable. Example:
-```
-ENABLE_DROPWIZARD_METRICS=true
-```
+3. Add the GDS metrics library to where your project is stored by adding:
+
+    Maven
+    ```
+    <repositories>
+        <repository>
+            <id>reliability-engineering-repository</id>
+            <name>Repository containing reliability enginieering dependencies</name>
+            <url>https://dl.bintray.com/reliability-engineering-gds/gds_metrics_dropwizard</url>
+        </repository>
+    </repositories>
+    ```
+
+    Gradle
+    ```
+    repositories {
+        maven { url "https://dl.bintray.com/reliability-engineering-gds/gds_metrics_dropwizard" }
+    }
+    ```
+
+4. Add your library as a dependency to your project.
 
-### Changes in your project
+    Maven
+    ```
+    <dependency>
+        <groupdId>engineering.gds-reliability</groupdId>
+        <artifactId>gds-metrics-dropwizard</artifactId>
+        <version>1.0.0</version>
+    </dependency>
+    ```
+
+    Gradle
+    ```
+    implementation 'engineering.gds-reliability:gds-metrics-dropwizard:1.0.0'
+    ```
+
+5. When your app is deployed to the PaaS, you can add an environment variable so Prometheus can discover your app’s metrics, for example:
+
+    ```
+    PROMETHEUS_METRICS_PATH=/metrics
+    ```
+
+## How to Change your project
 
 1. Add the `MetricsBundle`.
-```
-public void initialize(final Bootstrap<ExampleConfiguration> bootstrap) {
-    ...
-    bootstrap.addBundle(new MetricsBundle());
-    ...
-}
-```
+    ```
+    public void initialize(final Bootstrap<ExampleConfiguration> bootstrap) {
+        ...
+        bootstrap.addBundle(new MetricsBundle());
+        ...
+    }
+    ```
 
-2. Disable app security for the metrics path.
-If your application already has security (authentication) implemented, the metrics path defined in the variable `PROMETHEUS_METRICS_PATH` should be excluded because this path already has its own security. `Configuration.getInstance.getPrometheusMetricsPath()` can be used to recover the proper value.
+2. Disable app security for the metrics path. If your application already has security (authentication) implemented, the metrics path defined in the variable `PROMETHEUS_METRICS_PATH` should be excluded because this path already has its own security. `Configuration.getInstance.getPrometheusMetricsPath()` can be used to recover the proper value.
```

Comment:
is this the only type of security authentication? or is it that this is the preferred option? what are we auth'ing? the user to use the app? the app to use the paas?